tests/bin/eslint.js

```diff
@@ -122,21 +122,35 @@ describe("bin/eslint.js", () => {
             return assertExitCode(child, 1);
         });
 
-        it("gives a detailed error message if no config file is found in / or in ~/", () => {
-
-            // Assumes the root and home directories have no .eslintrc file.
-            const child = runESLint(["--stdin"], { cwd: "/" });
-
-            const exitCodePromise = assertExitCode(child, 1);
-            const stdoutPromise = getOutput(child).then(output => {
-                assert.match(output.stderr, /ESLint couldn't find a configuration file/);
-            });
+        it(
+            "gives a detailed error message if no config file is found in /",
+            () => {
+                if (
+                    fs.readdirSync("/").filter(
```

Comment:
Nitpick: I think it would be better to use `.some` here instead of `.filter`:

```js
if (fs.readdirSync("/").some(fileName => /^\.eslintrc(?:\.(?:js|yaml|yml|json))?$/.test(fileName))) {
    // ...
}
```

Reply:
Of course. Obvious, now that you mention it. Will do.

Reply:
Done.